eip-0039.md

```diff
@@ -0,0 +1,41 @@
+# EIP-39 Monotonic box creation height rule
+
+* Author: MrStahlfelge
+* Status: Proposed
+* Created: 05-Oct-2022
+* Implemented: no
+* Last edited: 05-Oct-2022
+* License: CC0
+* Forking: not needed for current draft 
```

Comment:
Soft-fork actually (new nodes will reject transactions breaking monotonic height rule, old nodes will accept both kinds of transactions).